templates/password_reset.txt

```diff
@@ -2,7 +2,7 @@
 {{t "Your Firefox Account password has been successfully reset." }}
 
 {{{t "This is an automated email; if you didn’t reset the password to your Firefox Account, you should reset it immediately at %(resetLink)s." }}}
-{{t "For more information, please visit %(supportUrl)s" }}
+{{{t "For more information, please visit %(supportUrl)s" }}}
```

Comment:
Can you add a space above this line? With the new lengthy reset link, this looks squashed.